Sources/Edge.swift

```diff
@@ -31,6 +30,7 @@ public class Edge: NSObject, Extension {
 
     public required init(runtime: ExtensionRuntime) {
         self.runtime = runtime
+        NetworkResponseHandler.currentPrivacyStatus = EdgeConstants.DEFAULT_PRIVACY_STATUS
```

Comment:
This is a bit redundant with the definition in the NetworkResponseHandler.
`static var currentPrivacyStatus: PrivacyStatus = EdgeConstants.DEFAULT_PRIVACY_STATUS`